runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePanel.java

```diff
@@ -37,18 +45,49 @@
 {
 	private static final int MAX_OFFERS = 8;
 
+	@Getter
+	private GrandExchangeSearchPanel grandExchangeSearchPanel;
+
 	private GrandExchangeOfferSlot[] offerSlotPanels = new GrandExchangeOfferSlot[MAX_OFFERS];
 
+	private JPanel buttonPanel = new JPanel();
+	private JPanel offerPanel = new JPanel();
+	private JPanel searchPanel = new JPanel();
+
+	private JButton offerButton = new JButton("Offers");
+	private JButton searchButton = new JButton("Search");
+
 	@Inject
-	GrandExchangePanel(ItemManager itemManager)
+	GrandExchangePanel(Client client, ItemManager itemManager, ScheduledExecutorService executor)
 	{
-		setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+		setLayout(new BorderLayout());
+
+		// Button Panel
+		buttonPanel.setLayout(new GridLayout(1, 2, 5, 0));
+		buttonPanel.setBorder(new EmptyBorder(0, 0, 5, 0));
+
+		offerButton.addActionListener(e -> showOffers());
```

Comment:
Can these buttons be changed to https://docs.oracle.com/javase/tutorial/uiswing/components/tabbedpane.html?That would be a lot cleaner approach to basically do tabbed panes in Swing :P